templates/deploy/params

```diff
@@ -0,0 +1,3 @@
+NAMESPACE=infographic-dev
```

Comment:
Could we add 3 sets of example params - i.e.: aligned with the Jenkinsfile, `dev`, `test`, `uat`?